format/src/main/thrift/carbondata.thrift

```diff
@@ -136,7 +136,7 @@ struct DataChunk2{
 * Represents a chunk of data. The chunk can be a single column stored in Column Major format or a group of columns stored in Row Major Format.
 **/
 struct DataChunk3{
-    1: required list<DataChunk2> data_chunk_list; // list of data chunk
+    1: required list<binary> data_chunk_list; // list of data chunk
```

Comment:
I guess it is better to keep `DataChunk2` to get clear dependency and understanding. Otherwise user cannot figure out what is inside `list<binary>`